content/pt/docs/concepts/cluster-administration/networking.md

```diff
@@ -0,0 +1,250 @@
+
+<!-- body -->
+
+Kubernetes é basicamente o compartilhamento de máquinas entre aplicações. Tradicionalmente,
+compartilhar máquinas requer a garantia de que duas aplicações não tentem utilizar
+as mesmas portas. Coordenar a alocação de portas entre múltiplos desenvolvedores é 
+muito dificil de fazer em escala e expõe os usuários a problemas em nível do cluster e 
+fora de seu controle.
+
+A alocação dinâmica de portas trás uma série de complicações para o sistema - toda
+aplicação deve obter suas portas através de flags de configuração, os servidores de API 
+devem saber como inserir números dinämicos de portas nos blocos de configuração, serviços
+precisam saber como buscar um ao outro, etc. Ao invés de lidar com isso, o Kubernetes 
+faz de uma maneira diferente.
+
+## O modelo de conectividade e rede do Kubernetes
+
+Todo `Pod` obtém seu próprio endereço IP. Isso significa que vocë não precisa 
+criar links explícitos entre os `Pods` e vocë quase nunca terá que lidar com o
+mapeamento de portas de contêineres para portas do host. Isso cria um modelo simples, 
+retro-compatível onde os `Pods` podem ser tratados muito mais como VMs ou hosts 
+físicos da perspectiva de alocação de portas, nomes, descobrimento de serviços 
+(_service discovery_), balanceamento de carga, configuração de aplicações e migrações.
+
+O Kubernetes impõe as seguintes requisitos fundamentais para qualquer implementação de 
+rede (exceto qualquer política de segmentação intencional):
+   * pods em um nó podem se comunicar com todos os pods em todos os nós sem usar _NAT_.
+   * agentes em um nó (por exemplo o kubelet ou um serviço local) podem se comunicar com 
+     todos os Pods naquele nó.
+
+Nota: Para as plataformas que suportam `Pods` executando na rede do host (como o Linux):
+
+   * pods alocados na rede do host de um nó podem se comunicar com todos os pods
+     em todos os nós sem _NAT_.
+
+Esse modelo não só é menos complexo, mas é principalmente compatível com o 
+desejto do Kubernetes de permitir a portabilidade com baixo esforço de aplicações 
+de VMs para contêineres. Se a sua aplicação executava anteriormente em uma VM, sua VM
+possuía um IP e podia se comunicar com outras VMs no seu projeto. Esse é o mesmo 
+modelo básico.
+
+Os endereços de IP no Kubernetes existem no escopo do `Pod` - contêineres em um `Pod`
+compartilham o mesmo _network namespace_ - incluíndo seu endereço de IP e MAC.
+Isso significa que contêineres que compõem um `Pod` podem se comunicar entre eles 
+através do endereço `localhost` e respectivas portas. Isso também significa que
+contêineres em um mesmo `Pod` devem coordenar a alocação e uso de portas, o que não 
+difere do modelo de processos rodando dentro de uma mesma VM. Isso é chamado de 
+modelo "IP-por-pod".
+
+Como isso é implementado é um detalhe do agente de execução de contêiner em uso.
+
+É possível solicitar uma porta no `Nó` que será encaminhada para seu `Pod` (chamado 
```

Comment:
```suggestion
É possível solicitar uma porta no `Node` que será encaminhada para seu `Pod` (chamado 
```
?

Reply:
The word “nó” is fine if not in backticks. Backticks imply something you literally type.